projects/aca-folder-rules/src/lib/rule-details/conditions/rule-simple-condition.ui-component.spec.ts

```diff
@@ -116,4 +129,24 @@ describe('RuleSimpleConditionUiComponent', () => {
     const unknownOptionMatOption = getByDataAutomationId('unknown-field-option');
     expect(unknownOptionMatOption).toBeNull();
   });
+
+  it('should provide select option when mimeType is selected and value filled', () => {
+    fixture.componentInstance.writeValue(mimeTypeMock);
+    fixture.detectChanges();
+
+    expect(getByDataAutomationId('simple-condition-value-select')).toBeTruthy();
+    expect(getByDataAutomationId('simple-condition-value-select').nativeElement.value).not.toBe('');
```

Comment:
instead of not.toBe('') it would be better to test exact value which is assigned. Now it passes always when value is not empty string but it does not test what value is set in reality. Someone can introduce a bug which sets incorrect value and it won't be detected by that test.